### ISSUE TITLE
The project needs a license.

### ISSUE THREAD

**Issue by Monnoroch** [shyiko/klob#1]
A license is required for using any sort of third party code in commercial context (basically in any company).

**Comment by thecoden** [comment]
I agree with this. We have forked this project to help [ktlint](https://github.com/pinterest/ktlint/issues/942), but I'm sure they would appreciate a clear license added as well.
